subsys/net/ip/Kconfig

```diff
@@ -148,6 +148,15 @@ config NET_TCP_ACK_TIMEOUT
 	various TCP states. The value is in milliseconds. Note that
 	having a very low value here could prevent connectivity.
 
+config NET_TCP_INIT_RETRANSMISSION_TIMEOUT
```

Comment:
I find it a bit strange that developer-facing option is named "CONFIG_NET_TCP_INIT_RETRANSMISSION_TIMEOUT", while user-facing description says "value of RTO". Suggestion: name the option CONFIG_NET_TCP_INITIAL_RTO, and for description, add "value of TCP RTO", so a user at least know what to google for.

Reply:
@pfalcon This makes it consistent with previous definition NET_TCP_ACK_TIMEOUT so it should end with timeout to indicate that this is a timeout. I can reword help description instead. I am sure user would be able to find it anyway, otherwise he probably does not need it ;)

Reply:
Up to you, those were just notes/suggestions with a "random user" hat on ;-)